tigrex/__main__.py

```diff
@@ -61,40 +64,61 @@ def search(*args):
         Card.__print_card_header(response, card_layout)
         Card.__print_card_search(response, card_layout)
 
+    @staticmethod
+    def currency(*args):
+        if not len(args) == 1:
+            return "Please only provide one value for the currency"
+        if args[0] == 'eur':
+            config['Price']['sign'] = '€'
+            config['Price']['Cur'] = str(args[0])
+        elif args[0] == 'usd':
+            config['Price']['sign'] = '$'
+            config['Price']['Cur'] = str(args[0])
+        elif args[0] == 'tix':
+            config['Price']['sign'] = 'tix '
+            config['Price']['Cur'] = str(args[0])
+        else:
+            return "Please only provide, usd, eur or tix as currency options"
+
+        with open('config.ini', "w") as configfile:
+            config.write(configfile)
+
     @staticmethod
     def __get_card_description(response, card_layout):
         description = EMPTY_STRING
 
         if card_layout == NORMAL or card_layout == MELD or card_layout == SAGA:
             if CREATURE in response.json()[TYPE_LINE]:
                 description = response.json()[TYPE_LINE] + NEW_LINE + NEW_LINE + \
-                              response.json()[ORACLE_TEXT] + NEW_LINE + NEW_LINE + \
-                              response.json()[POWER] + SPACE + FORWARD_SLASH + SPACE + response.json()[TOUGHNESS]
+                    response.json()[ORACLE_TEXT] + NEW_LINE + NEW_LINE + \
```

Comment:
@Shadesfear, is there a reason why line breaks were added on certain lines?

Reply:
This has to do with my linter that makes sure lines are not longer than 79, according to pep8. It's not needed. But I guess those who follow pep8 likes it :)

Reply:
I see that you already fixed the issues :) So the MR can be closed